controllers/storagecluster/storagecluster_controller_test.go

```diff
@@ -158,9 +159,9 @@ func getMockDeviceSets(name string, count int, replica int, portable bool) []api
 			Name:                name,
 			Count:               count,
 			Replica:             replica,
-			DataPVCTemplate:     mockDataPVCTemplate,
-			MetadataPVCTemplate: mockMetaDataPVCTemplate,
-			WalPVCTemplate:      mockWalPVCTemplate,
+			DataPVCTemplate:     *mockDataPVCTemplate.DeepCopy(),
```

Comment:
Why did you make it a pointer? Is it required to fix unit test?

Reply:
The DeepCopy() returns a pointer, but we don't need a pointer, hence we use the pointer value by `*`